pycytominer/cyto_utils/cells.py

```diff
@@ -469,6 +475,7 @@ def load_compartment(self, compartment):
     def aggregate_compartment(
         self,
         compartment,
+        features,
```

Comment:
maybe worth making `features='infer'` as default to retain previous behavior and not require additional arguments if someone uses a new version?

Reply:
See other comment, but my assumption when writing this was that we don't want the user changing the features after initializing SingleCells; if so, we don't want a default here.

Reply:
> we don't want the user changing the features after initializing SingleCells

I agree, but then why do we need this argument in `aggregate_compartment` at all?